janusgraph-dist/README.md

```diff
@@ -31,34 +31,6 @@ The documentation output appears in:
 
 * site/
 
-## Building deb/rpm packages
```

Comment:
I didn't try to build dep/rpm packages but why does this functionality removed?

Reply:
I went over the functionality, it is massively outdated.

If we want to have such functionality, it should be tested and update. 
(I expected we have to rebuild from stretch. => Currently, the functionality completely independent from our normal release which would require to double all scripts for both zip and system packages.)